main/wallet.js

```diff
@@ -84,7 +84,15 @@ function getBalance () {
  * @returns {string}
  */
 function getScheduledRewards () {
-  return ethers.utils.formatUnits(scheduledRewards, 18)
+  const fullPrecision = ethers.utils.formatUnits(scheduledRewards, 18)
+  const [whole, fraction] = fullPrecision.split('.')
+  if (fraction === undefined) return fullPrecision
+  const truncated = fraction
+    // keep the first 6 digits, discard the rest
+    .slice(0, 6)
+    // remove trailing zeroes
+    .replace(/0+$/, '0')
+  return [whole, truncated].join('.')
```

Comment:
would be great to add unit tests for this, this kind of function lends itself perfectly (you want to check many edge cases)

Reply:
Great suggestion! 😍 I should have thought about this myself. 🙈 

Here is the PR; the tests helped me to discover a bug.
https://github.com/filecoin-station/desktop/pull/1100